src/dir.rs

```diff
@@ -705,11 +705,11 @@ where
 
                 match _get_dir_content(_path, depth) {
                     Ok(items) => {
-                        let mut _files = items.files;
-                        let mut _dirrectories = items.directories;
+                        let mut files = items.files;
```

Comment:
This is changes breaking a logic... files and files it's different variables. After remove "_" we get the append method for the new variable, but we should use old. For variable "directories" we get same result.

Reply:
I have reverted the changes.